components/landing/TheQuickStart/ProvidersList.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <section class="providers">
+    <h4>Providers</h4>
+    <ul class="providers__list">
+      <li v-for="item in providersList" :key="item.id">
+        <a class="providers__list__item" :href="item.link" target="_blank">
+          {{ item.title }}
+          <span class="providers__list__cta">
+            <span class="providers__list__cta-label">Learn more</span>
+            <Launch16 class="providers__list__cta-icon" />
+          </span>
```

Comment:
Have you tried to use here the `AppCta` component?

Reply:
For this particular piece, the whole "provider bar" _is_ the clickable link, not just this button on the right... 

So I decided not to use a button or CTA component... 


https://user-images.githubusercontent.com/6276074/223856140-9cecd9dc-ab5c-4e79-9f44-478d1d72096a.mov